options.lua

```diff
@@ -0,0 +1,69 @@
+local profile = Game.GetProfile()
+
+local function split_string(inputstr, sep)
+    if sep == nil then
+        sep = "%s"
+    end
+    local t = {}
+    for str in string.gmatch(inputstr, "([^" .. sep .. "]+)") do
+        table.insert(t, str)
+    end
+    return t
+end
+
+local function update_gui(menu, metric, value)
+    menu[metric].value = value
+    menu[metric .. '_text'].text = string.format("%.0f%%", value)
+end
+
+local Mod_layout =
+[[
+<VerticalList child_padding=8>
+    <Text fill=false text="Health" style=hl textalign=left />
+    <HorizontalList child_fill=true>
+        <Text text="0" fill=false valign=center textalign=right width=50 margin_right=5 id=health_text/>
+        <Slider height=36 min=0 max=100 step=1 id=health on_change={on_slider_changed}/>
+    </HorizontalList>
+
+    <Text fill=false text="Damage" style=hl textalign=left />
+    <HorizontalList child_fill=true>
+        <Text text="0" fill=false valign=center textalign=right width=50 margin_right=5 id=damage_text/>
+        <Slider height=36 min=0 max=100 id=damage on_change={on_slider_changed}/>
+    </HorizontalList>
+</VerticalList>
+]]
+
+return UI.New(Mod_layout, {
+    construct = function(menu)
+        if not profile.Mod_Weak_Bug then
```

Comment:
I think `Mod_Weak_Bug` is totally fine but just for reference I think `GetProfile()` returns a table that is specific to this mod in the first place.

GetProfile ([Game](https://modding.desyncedgame.com/syntax.html#game) Function)
```
Get the local profile table
If called without argument from a mod other than 'Main', will return a mod specific child table parent.mods[mod_id]
If called with an empty string or nil, will always return entire parent table

Argument 1: Mod id (OPTIONAL) (string)
Return Value: Profile table (table)
```